application/views/survey_responses_view.py

```diff
@@ -43,14 +43,17 @@ def add_survey_response(case_id):
         if not file_name:
             return make_response(MISSING_DATA, 404)
 
-        upload_success = survey_response.add_survey_response(case_id, file, file_name, survey_ref)
-
-        if upload_success == "False":
+        try:
+            log.info('Processing file')
+            survey_response.add_survey_response(case_id, file, file_name, survey_ref)
+        except FileTooSmallError:
+            log.error('File too small')
             return make_response(FILE_TOO_SMALL, 400)
-
-        if upload_success:
-            return make_response(UPLOAD_SUCCESSFUL, 200)
-        else:
+        except SurveyResponseError:
+            log.error('Something went wrong')
```

Comment:
You probably want a more descriptive logging message.

Though if you add it in the above method, it's probably not needed here too.